runtime/layers/fpm-dev/Dockerfile

```diff
@@ -4,6 +4,20 @@ FROM $LAYER_IMAGE
 # Override the config so that PHP-FPM listens on port 9000
 COPY php-fpm.conf /opt/bref/etc/php-fpm.conf
 
+USER root
+ENV current_os=linux
+RUN version=$(php -r "echo PHP_MAJOR_VERSION.PHP_MINOR_VERSION;")-zts \
+    && curl -A "Docker" -o /tmp/blackfire-probe.tar.gz -D - -L -s https://blackfire.io/api/v1/releases/probe/php/$current_os/amd64/$version \
+    && mkdir -p /tmp/blackfire \
+    && tar zxpf /tmp/blackfire-probe.tar.gz -C /tmp/blackfire \
+    && cp /tmp/blackfire/blackfire-*.so $(php -r "echo ini_get('extension_dir');")/blackfire.so \
+    && rm -rf /tmp/blackfire /tmp/blackfire-probe.tar.gz
+
+# Build the final image from the lambci image that is close to the production environment
+FROM lambci/lambda:provided
```

Comment:
Is there a reason for this addition? I fear this will break features in the dev image.

Reply:
OK one advantage is that it provides a final image with only 1 layer (lighter to download).

Could you change the comment on the line above to:

```
# Build the final image from scratch so that it is composed only from 1 layer (smaller image)
```